plugins/lookup/lookup.py

```diff
@@ -376,12 +378,9 @@ def get_secret_data(
         )
 
         try:
-            if not state_file_dir:
-                client.access_token_login(access_token.str)
-            else:
-                create_state_dir(state_file_dir)
-                state_file = str(Path(state_file_dir, access_token.access_token_id))
-                client.access_token_login(access_token.str, state_file)
+            create_state_dir(state_file_dir)
+            state_file = str(Path(state_file_dir, access_token.access_token_id).expanduser())
```

Comment:
could we maybe re-use the expanded path `state_dir` from `create_state_dir` (return it), so the only thing we would have to do then is to just join `access_token.access_token_id` ?
```suggestion
            state_dir = create_state_dir(state_file_dir)
            state_file = str(state_dir / access_token.access_token_id))
```

Reply:
Ah yeah. That's much better 😁 

Fixed in 48b519feaa60a9210668a1061b27b9e4f08eb8f7.